lib/mongoid/fields.rb

```diff
@@ -573,6 +574,8 @@ def field_for(name, options)
       def unmapped_type(options)
         if "Boolean" == options[:type].to_s
           Mongoid::Boolean
+        elsif "StringifiedSymbol" == options[:type].to_s
```

Comment:
Am I correct that all 3 of the following work:

```
field :foo, type: :stringified_symbol
field :foo, type: 'StringifiedSymbol'
field :foo, type: Mongoid::Extensions::StringifiedSymbol
```

If yes, I think the second use isn't really needed given the first and the third ones.

Reply:
I believe we discussed removing this change?